src/redis-cli.c

```diff
@@ -87,6 +87,9 @@
     "[ERR] Invalid arguments: you need to pass either a valid " \
     "address (ie. 120.0.0.1:7000) or space separated IP " \
     "and port (ie. 120.0.0.1 7000)\n"
+#define CLUSTER_MANAGER_INVALID_HOST_ARG_NEW \
+    "[ERR] Invalid arguments: you need to pass a valid " \
+    "address (ie. 120.0.0.1:7000)\n"
```

Comment:
```suggestion
    "[ERR] Invalid arguments: you need to pass a valid " \
    "address:port pair (e.g. 120.0.0.1:7000)\n"
```

Reply:
In fact there is only on choice. So maybe we can just remove it?

Reply:
Remove the example or what?